corehq/apps/sso/backends.py

```diff
@@ -84,6 +84,7 @@ def authenticate(self, request, username, idp_slug, is_handshake_successful):
         if not is_new_user and not web_user.is_active:
             web_user.is_active = True
             web_user.save()
+            user.refresh_from_db()
```

Comment:
Just acknowledging that in the interest of limiting the blast radius of this change, this feels like the right place to put the refresh. As Jing explained, the couch user save will trigger a synchronous sync to the Django user, so refreshing the Django user right after seems appropriate.